lib/nx_zlib.c

```diff
@@ -932,7 +934,8 @@ void nx_hw_init(void)
 	nx_config.per_job_len = (1024 * 1024); /* less than suspend limit */
 	nx_config.strm_def_bufsz = (1024 * 1024); /* affect the deflate fifo_out */
 	nx_config.soft_copy_threshold = 1024; /* choose memcpy or hwcopy */
-	nx_config.compress_threshold = (10*1024); /* collect as much input */
+	nx_config.cache_threshold = 8 * 1024; /* Cache input before
+					       *  processing */
```

Comment:
Why change to 8KB instead of 10KB?

Reply:
Ideally, we want to cache as little as possible and provide the highest throughput.
In my tests, caching 8KiB or 10KiB had the same performance.